text/0003-encrypted-memos.md

```diff
@@ -1,7 +1,6 @@
 - Feature Name: Encrypted Memos
 - Start Date: (2021-06-18)
 - MCIP PR: [mobilecoinfoundation/mcips#3](https://github.com/mobilecoinfoundation/mcips/pull/3)
-- MobileCoin Epic: None
```

Comment:
we could link to the PR I guess: https://github.com/mobilecoinfoundation/mobilecoin/pull/814